lib/active_scaffold/extensions/action_view_rendering.rb

```diff
@@ -149,8 +149,20 @@ def render_embedded(options)
           model = remote_controller.to_s.sub(%r{.*/}, '').singularize
           content_tag(:div, :class => 'active-scaffold-header') do
             content_tag(:h2) do
-              link_label = options[:label] || active_scaffold_config_for(model).list.label
-              link_to(link_label, url, remote: true, class: 'load-embedded', data: {error_msg: as_(:error_500)}) <<
+              label = options[:label] || begin
+                # attempt to retrieve the active_scaffold_config by constantizing
+                # the controller path
+                controller_config = "#{remote_controller}_controller".camelize.constantize.active_scaffold_config rescue nil
+                controller_config.try(:label) || begin
+                  # if we couldn't determine the controller config by instantiating the
+                  # controller class, parse the ActiveRecord model name from the
+                  # controller path, which might be a namespaced controller (e.g., 'admin/admins')
+                  model = remote_controller.to_s.sub(%r{.*/}, '').singularize
```

Comment:
Why does this need to be re-defined if it was already defined on line 149? Maybe just left over from some testing?

Reply:
```suggestion
```